examples/todomvc-main/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@feature-hub/todomvc-main",
+  "version": "0.0.0",
+  "private": true,
+  "license": "MIT",
+  "main": "dist/index.js",
+  "scripts": {
+    "prebundle": "rimraf dist",
+    "bundle": "parcel build src/index.tsx"
+  },
+  "dependencies": {
+    "@feature-hub/core": "^0.2.0",
+    "@feature-hub/react": "^0.2.0",
+    "@feature-hub/todomvc-todo-manager": "^0.0.0",
+    "styled-components": "^4.1.2"
+  },
+  "devDependencies": {
+    "@types/react": "^16.7.8",
+    "@types/styled-components": "^4.1.2",
+    "parcel-bundler": "^1.10.3",
+    "react": "^16.6.3",
```

Comment:
why is react a dev dependency, but not lit-html in the header?

Reply:
It was originally because `react` was declared as external dependency in the webpack config. We should probably do the same with rollup.

Reply:
Well, [it's complicated](https://github.com/parcel-bundler/parcel/issues/144). I would suggest switching back to webpack.

Reply:
We'll tackle this in #97.